indico/modules/events/abstracts/blueprint.py

```diff
@@ -42,6 +44,8 @@
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
                  RHAbstractsDownloadAttachment)
+_bp.add_url_rule('/abstracts/mine', 'my_abstracts', RHMyAbstracts)
+_bp.add_url_rule('/abstracts/my-abstracts.pdf', 'my_abstracts_pdf_export', RHMyAbstractsExportPDF)
```

Comment:
Sure, but we have tons of other enpoints with the `_export`. Only in contributions we have:

* `contributions_pdf_export`
* `contributions_pdf_export_book`
* `contributions_pdf_export_book_sorted`
* `export_pdf`